src/main/clojure/aerospike_clj/utils.clj

```diff
@@ -21,7 +21,8 @@
   (= bin-names [""]))
 
 (defn string-keys?
-  "Predicate function to determine whether all keys provided for bins are strings."
+  {:docstring  "Predicate function to determine whether all keys provided for bins are strings."
```

Comment:
```suggestion
  {:doc  "Predicate function to determine whether all keys provided for bins are strings."
```

`:doc` is the correct metadata key that Cursive picks up to generate doc tooltips for Vars.

Reply:
Fixed